bitcoind-rpc/src/main/scala/org/bitcoins/rpc/serializers/JsonReaders.scala

```diff
@@ -498,6 +498,41 @@ object JsonReaders {
       }
 
   }
+  implicit object AnalyzePsbtInputReads extends Reads[AnalyzePsbtInput] {
+    override def reads(json: JsValue): JsResult[AnalyzePsbtInput] =
+      for{
+      hasutxo <- (json \ "has_utxo").validate[Boolean]
+      isfinal <- (json \ "is_final").validate[Boolean]
+      missing <- (json \ "missing").validateOpt[AnalyzePsbtMissing]
+      next <- (json \ "next").validateOpt[String]
+
+      }
+      yield {
+        AnalyzePsbtInput(
+          hasutxo = hasutxo,
+          isfinal = isfinal,
+          missing = missing,
+          next = next
+        )
+      }
+  }
+  implicit object AnalyzePsbtMissingReads extends Reads[AnalyzePsbtMissing] {
+    override def reads(json: JsValue): JsResult[AnalyzePsbtMissing] =
+      for{
+      pubkeys <- (json \ "pubkeys").validateOpt[Vector[ECPublicKey]]
+      signatures <- (json \ "signatures").validateOpt[Vector[String]]
+      redeemscript <- (json \ "redeemscript").validateOpt[String]
+      witnessscript <- (json \ "witnessscript").validateOpt[String]
+      }
+      yield{
+        AnalyzePsbtMissing(
+          pubkeys = pubkeys,
+          signatures = signatures,
+          redeemscript = redeemscript,
+          witnessscript = witnessscript
+        )
+      }
+  }
```

Comment:
You should be able to replace all of these two objects by just going to JsonSerializers and having:
```
implicit val analyzePsbtMissingReads: Reads[AnalyzePsbtMissing] =
  Json.reads[AnalyzePsbtMissing]

implicit val analyzePsbtInputReads: Reads[AnalyzePsbtInput] =
  Json.reads[AnalyzePsbtInput]

implicit val analyzePsbtResultReads: Reads[AnalyzePsbtResult] =
  Json.reads[AnalyzePsbtResult]
```